pyproject.toml

```diff
@@ -3,3 +3,4 @@ requires = ["setuptools>=42", "setuptools_scm[toml]>=3.4", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
+write_to = "src/stdatamodels/_version.py"
```

Comment:
Is there a reason to write this file out?  In the past, we've just had code in `__init__.py` that sets it using `pkg_resources`.  I.e. for `jwst`:

https://github.com/spacetelescope/jwst/blob/c04bc7e1bcd7cb96c34308771a89a5e39abccf49/jwst/__init__.py#L8-L11

Thought guess letting `setuptools_scm` do all the work does make `__init__.py` for the package a bit simpler.  Is there any downside in writing out a `_version.py` file?

Reply:
I'm not sure if there are any downsides, this is the setuptools_scm documentation on that option:

```
A path to a file that gets replaced with a file containing the current version. It is ideal for creating a version.py file within the package, typically used to avoid using pkg_resources.get_distribution (which adds some overhead).
```

and astropy uses this feature: https://github.com/astropy/astropy/blob/master/setup.py#L69